src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/System.Xml.XmlSchemaSet.Tests.csproj

```diff
@@ -52,4 +53,4 @@
     </SupplementalTestData>
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
at the end of the file make sure to have an empty newline. See here: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file